app/src/main/java/com/example/minutejournal/ui/main/DTO/EntriesDTO.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.minutejournal.ui.main
+package com.example.minutejournal.ui.main.DTO
```

Comment:
While DTO is an acronym, Java/Kotlin conventions are that package names are lowercase... so I'd make this lowercase.

Reply:
None the less, a good suggestion... good place to organize data classes.